src/web/url/origin.md

```diff
@@ -66,7 +64,7 @@ fn run() -> Result<()> {
     Ok(())
 }
 #

```

Comment:
please remove the trailing `#` and all uses of `error_chain` when not completely necessary.